packages/scss/src/components/util/index.scss

```diff
@@ -147,6 +143,14 @@
 	@include text.ellipsis('!important');
 }
 
+.u-fontWeightRegular {
```

Comment:
Since `font-weight` properties are already listed in core, it seem unconsistant to find those new here, even if their syntax is slightly different.

Maybe we could do something like  
`$fontWeight: '200', '300', '400', '500', '600', '700', '800', '900', 'normal', 'bold', 'regular:400', 'semiBold:600';` in core and add some logic in `core.classes` ? That doesn't sound too complex

Reply:
My opinion here: https://github.com/LuccaSA/lucca-front/issues/1813#issuecomment-1237944561